.circleci/config.yml

```diff
@@ -125,6 +124,14 @@ references:
           name: Install Codacy
           command: make -C .doks/ deploy_to_doks_from_chartmuseum RELEASE=$RELEASE NAMESPACE=$NAMESPACE VERSION=$(cat .version) HELM_REPOSITORY=$HELM_REPOSITORY
 
+  rotate_pods: &rotate_pods
```

Comment:
can you put this right after the helm upgrade? on the makefile?
Its stays just in one place and its easier to delete later

Reply:
sure! :) that's actually much simpler!